7/GrainUnitTests/GrainUnitTests.csproj

```diff
@@ -27,10 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Client\Client.csproj" />
-    <ProjectReference Include="..\..\..\Grains\Grains.csproj" />
-    <ProjectReference Include="..\..\..\Interfaces\Interfaces.csproj" />
-    <ProjectReference Include="..\..\..\Library\Library.fsproj" />
+    <ProjectReference Include="..\Grains\Grains.csproj" />
   </ItemGroup>
 
 </Project>
```

Comment:
The project evolved significantly since the last time we dealt with unit tests and now individual unit testing projects look like they could be more aligned:

![Screen Shot 2021-01-06 at 2 48 18 PM](https://user-images.githubusercontent.com/6170837/103783466-aca3cb80-5030-11eb-8bbe-f76608dd80c5.png)

I'd rename:

* `GrainUnitTests` -> `Grains.UnitTests`
* `UnitTests` -> `Library.UnitTests`